server/routes/induction/create/reasonsNotToGetWorkCreateController.test.ts

```diff
@@ -0,0 +1,202 @@
+      flash: jest.fn(),
+    } as unknown as Request
+
+    res = {
+      redirect: jest.fn(),
+      render: jest.fn(),
+    } as unknown as Response
+  })
+
+  describe('getReasonsNotToGetWorkView', () => {
+    it('should get the reasons not to get work create view', async () => {
+      // Given
+      const prisonerSummary = aValidPrisonerSummary()
+      req.session.prisonerSummary = prisonerSummary
+
+      const inductionDto: InductionDto = undefined
+      req.session.inductionDto = inductionDto
+
+      req.session.reasonsNotToGetWorkForm = undefined
+
+      const expectedReasonsNotToGetWorkForm: ReasonsNotToGetWorkForm = {
+        reasonsNotToGetWork: [],
+      }
+
+      const expectedView = {
+        prisonerSummary,
+        form: expectedReasonsNotToGetWorkForm,
+        backLinkUrl: '/prisoners/A1234BC/create-induction/hoping-to-work-on-release',
+        backLinkAriaText: `Back to Is Jimmy Lightfingers hoping to get work when they're released?`,
+        errors: noErrors,
+      }
+
+      // When
+      await controller.getReasonsNotToGetWorkView(req, res, next)
+
+      // Then
+      expect(res.render).toHaveBeenCalledWith('pages/induction/reasonsNotToGetWork/index', expectedView)
+      expect(req.session.hopingToWorkOnReleaseForm).toBeUndefined()
+      expect(req.session.inductionDto).toEqual(inductionDto)
+    })
+
+    it('should get the reasons not to get work create view with form data', async () => {
+      // Given
+      const prisonerSummary = aValidPrisonerSummary()
+      req.session.prisonerSummary = prisonerSummary
+
+      const inductionDto: InductionDto = undefined
+      req.session.inductionDto = inductionDto
+
+      const expectedReasonsNotToGetWorkForm: ReasonsNotToGetWorkForm = {
+        reasonsNotToGetWork: [ReasonNotToGetWorkValue.HEALTH, ReasonNotToGetWorkValue.LIMIT_THEIR_ABILITY],
+      }
+      req.session.reasonsNotToGetWorkForm = expectedReasonsNotToGetWorkForm
+
+      const expectedView = {
+        prisonerSummary,
+        form: expectedReasonsNotToGetWorkForm,
+        backLinkUrl: '/prisoners/A1234BC/create-induction/hoping-to-work-on-release',
+        backLinkAriaText: `Back to Is Jimmy Lightfingers hoping to get work when they're released?`,
+        errors: noErrors,
+      }
+
+      // When
+      await controller.getReasonsNotToGetWorkView(req, res, next)
+
+      // Then
+      expect(res.render).toHaveBeenCalledWith('pages/induction/reasonsNotToGetWork/index', expectedView)
+      expect(req.session.hopingToWorkOnReleaseForm).toBeUndefined()
+      expect(req.session.inductionDto).toEqual(inductionDto)
+    })
+  })
+
+  describe('submitReasonsNotToGetWorkForm', () => {
+    it('should redisplay page given form is submitted with validation errors', async () => {
+      // Given
+      const prisonerSummary = aValidPrisonerSummary()
+      req.session.prisonerSummary = prisonerSummary
+      const inductionDto = { prisonNumber } as InductionDto
```

Comment:
As above, I think your induction on the session needs the properties that would have been set on the previous screen